### PR TITLE
✨ xdai support

### PR DESCRIPTION
**Pull request by nionis**

### CODE REVIEW

File: commands/abstractCommand.ts
Comment (peterbraden) [inline commands/abstractCommand.ts:5]: This actually won't work - it needs the empty string to know that the result is empty, rather than all options.

File: commands/index.ts
Comment (peterbraden) [inline commands/index.ts:105]: some autocompletion is async?

File: commands/abstractCommand.ts
Comment (nionis) [reply commands/abstractCommand.ts:5]: how can I reproduce a scenario like that? is this correct `ping foo`?

File: commands/index.ts
Comment (nionis) [reply commands/index.ts:105]: yeah, but `this.find` is not

https://github.com/hoprnet/hopr-chat/blob/7c64d9ca1e31b3182270cc2f377e329abf1546b3/commands/index.ts#L76

Comment (peterbraden) [reply commands/index.ts:105]: ah, yeah, my bad, misread this,

File: commands/abstractCommand.ts
Comment (peterbraden) [reply commands/abstractCommand.ts:5]: I think (although it's been a week since I looked at this) that the flow to reproduce is:
for ping foo

'Ping Fo' Tab -> shows ping foo
'Ping Fog' Tab -> No results

Comment (nionis) [reply commands/abstractCommand.ts]: reverted to using `[[''], line]`

**Review by peterbraden — APPROVED** [review]

**Comment by peterbraden** [comment]
DON'T MERGE THIS BRANCH

I'm gonna hack the github actions to do the npm publish with the xdai tag. We can back it out later.

**Review by peterbraden — CHANGES_REQUESTED** [review]
Need to back out github action changes before merge

**Review by peterbraden — APPROVED** [review]

**Pull request merged**
